src/components/Admin/Onboard/ProductAddModal.vue

```diff
@@ -86,38 +86,40 @@
   </div>
 </template>
 <script>
-  import {Toast, Loading} from 'quasar'
+  import {
+    Toast,
+    Alert
+  } from 'quasar'
   import axios from 'axios'
   import draggable from 'vuedraggable'
   import shop from '../../../api/shop'
-  const IMAGEUPLOAD = shop.API_URL + 'assets/image/upload'
   export default {
     props: ['current_category'],
     components: {
       draggable
     },
     methods: {
       onFileChange (e) {
+        var reader = new FileReader()
         var files = e.target.files || e.dataTransfer.files
         if (!files.length) return
-        var reader = new FileReader()
+        console.log(files)
+        if (files.length !== 1) {
```

Comment:
checking if the user clicked more then one file seems reasonable. We could pick the first one but if they dont realize they clicked more then one and dont look at the image they could fat finger it